implementation/common/pom.xml

```diff
@@ -51,6 +51,11 @@
       <groupId>org.jboss.logging</groupId>
       <artifactId>jboss-logging-processor</artifactId>
     </dependency>
+    <dependency>
```

Comment:
I've noticed we use 
```
<dependency>
      <groupId>junit</groupId>
      <artifactId>junit</artifactId>
      <scope>test</scope>
    </dependency>
```
in other modules - I don't mind introducing a jupiter dependency, but I guess it would make sense to have it consistent as part of the dedicated PR ? I suppose we can keep this dependency for this PR and then follow up...